app/views/pages/api_release_notes.html.haml

```diff
@@ -9,6 +9,43 @@
     Github issues
   for bugs or other problems.
 
+%section.api-documentation
+  %hr
+  %h2 7th March 2019
+  %ul.list-bullet
+    %li
+      Expose offence
+      %code.code='unique_code'
+      and optional filter on the
+      %a{ href: '/api/documentation?v=1#!/api/getApiOffences', target: '_blank' } api/offences
+      endpoint
+      %br/
+      %br/
+      To facilitate the lookup of offences the unique identifiers that
+      can be relied on not to change (i.e.
+      %code.code='unique_code'
+      ) have been exposed.
+      %br
+      %br
+      .panel
+        Note that an offence's
+        %code.code='id'
+        should not be persisted in consumers and relied on to add a specific offence to a claim. The reason for this is that internal id's could change unexpectedly. We will, however, endeavour to not change these for historical reasons. Instead an offence should be queried on this endpoint using the
```

Comment:
A cosmetic thing, but this line is very long!

Reply:
no worries, will break up. has no impact on the HTML but you are right it makes it easier to read in github and locally (where text editor wrapping normally handles it anyway)